### PR TITLE
chore(infrastructure): Add stylelint config for screenshot tests

### PR DESCRIPTION
**Pull request by acdvorak**
This PR:
- Adds `test/**/*.scss` to `npm run fix:css` and `npm run lint:css`
- Requires all Sass symbols (variables/functions/mixins) to begin with `test-` or `custom-` rather than `mdc-`
- Moves `ignoreFiles` from `.stylelintrc.yaml` to `.stylelintignore`

    According to the [stylelint configuration docs](https://github.com/stylelint/stylelint/blob/master/docs/user-guide/configuration.md#ignorefiles):
    > _Note that `ignoreFiles` is not an efficient method for ignoring lots of files._ If you want to ignore a lot of files efficiently, use `.stylelintignore` or adjust your files globs.
- Ignores all directories named `node_modules` or `out`, regardless of where they are
- Extends the root `.stylelintrc.yaml` file

### CODE REVIEW

File: test/screenshot/.stylelintrc.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  custom-property-pattern: ^(?:test|custom)-.+
+  selector-class-pattern:
+    - ^(?:test|custom)-.+
+    - resolveNestedSelectors: true
+  selector-id-pattern: ^(?:test|custom)-.+
+
+  # Sass
+  scss/dollar-variable-pattern:
+    - ^(?:test|custom)-.+
+    -
```
Comment (patrickrodee) [inline test/screenshot/.stylelintrc.yaml]: Is this an extra line?

File: .stylelintignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+**/node_modules
```
Comment (patrickrodee) [inline .stylelintignore]: Do we need another .styelintignore inside the `test/screenshot` directory or does this one handle all?

Comment (patrickrodee) [reply .stylelintignore]: Polite ping on this @advorak

File: test/screenshot/.stylelintrc.yaml (hunk 1 above)
Comment (acdvorak) [reply test/screenshot/.stylelintrc.yaml]: Removed!

File: .stylelintignore (hunk 2 above)
Comment (acdvorak) [reply .stylelintignore]: This root file handles all of them. IIRC `stylelint` doesn't support merging multiple `.stylelintignore` files.

**Pull request merged**